src/mol-plugin-state/transforms/model.ts

```diff
@@ -786,7 +786,7 @@ const StructureSelectionFromScript = PluginStateTransform.BuiltIn({
     from: SO.Molecule.Structure,
     to: SO.Molecule.Structure,
     params: () => ({
-        script: PD.Script({ language: 'mol-script', expression: '(sel.atom.atom-groups :residue-test (= atom.resname ALA))' }),
+        script: PD.Script({ language: 'pymol', expression: 'all' }),
```

Comment:
If we really want to we can add a PluginConfig options to change the default here.

Reply:
It would be excellent if we can set default language. 

By the way, it was so nice that you placed the language selector where it is now. We can switch languages and perform multi-language selection onto single structure in layer-by-layer manner. Multi-language operation in single viewer is new and interesting experience.